index.html

```diff
@@ -14,6 +14,73 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
-</body>
+
```

Comment:
Hmm. This looks weird... it's almost as if someone forgot to put in their meta viewport tag... 🤔

![image](https://user-images.githubusercontent.com/8473052/57825938-17730580-776f-11e9-9613-6edfe7a47008.png)